<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1243</width>
    <height>791</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <stylestrategy>PreferAntialias</stylestrategy>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">MainWindow{
border-image: url(:/resource/timg.jpg);
border-radius:15px
}
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>551</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox{
font: 15pt &quot;楷体&quot;;
border-radius:10px;
}</string>
    </property>
    <property name="title">
     <string>string类型数据</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>521</width>
       <height>38</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QComboBox" name="comboBox">
        <property name="styleSheet">
         <string notr="true">font: 15pt &quot;楷体&quot;;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="RadioFrist">
        <property name="styleSheet">
         <string notr="true">font: 15pt &quot;Consolas&quot;;</string>
        </property>
        <property name="text">
         <string>True</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="RadioSecond">
        <property name="styleSheet">
         <string notr="true">font: 15pt &quot;Consolas&quot;;</string>
        </property>
        <property name="text">
         <string>False</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="LineKey">
        <property name="styleSheet">
         <string notr="true">border-radius:5px;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>551</width>
      <height>591</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTextEdit" name="TextMain">
       <property name="styleSheet">
        <string notr="true">font: italic 18pt &quot;Consolas&quot;;
background-color: rgb(255, 251, 184);  
border-radius:5px;

</string>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustIgnored</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="ButtonImport">
         <property name="styleSheet">
          <string notr="true">font: 15pt &quot;楷体&quot;;
</string>
         </property>
         <property name="text">
          <string>导入</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ButtonClear">
         <property name="styleSheet">
          <string notr="true">font: 15pt &quot;楷体&quot;;</string>
         </property>
         <property name="text">
          <string>清空</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ButtonExport">
         <property name="styleSheet">
          <string notr="true">font: 15pt &quot;楷体&quot;;</string>
         </property>
         <property name="text">
          <string>导出</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="ButtonExit">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>0</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#queenMenuButton{
background-color: rgba(255, 255, 255, 0);
border:2px solid white;border-radius:10px;
}
 
QPushButton#queenMenuButton:hover{ 
background-color: rgba(255, 255, 255, 150);
border:2px solid white;border-radius:10px;
}
 
QPushButton#queenMenuButton:pressed{
background-color: rgba(255, 255, 255, 150);
border:2px solid white;border-radius:10px;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/resource/exit.ico</normaloff>:/resource/exit.ico</iconset>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="LabelTitle">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonAbout">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>0</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Consolas&quot;;
border:2px groove gray;
border-radius:10px;
padding:2px 4px;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>about</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>740</y>
      <width>421</width>
      <height>22</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>透明度</string>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>90</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>LabelTitle</zorder>
   <zorder>ButtonExit</zorder>
   <zorder>ButtonAbout</zorder>
   <zorder>horizontalSlider</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
